libs/remix-ui/workspace/src/lib/remix-ui-workspace.tsx

```diff
@@ -350,6 +350,10 @@ export function Workspace () {
     )
   }
 
+  const formatNameForReadonly = (name: string) => {
+    return global.fs.readonly ? name + " (read-only)" : name
```

Comment:
from where does `global.fs` come from?

Reply:
it's a file system we use. you mean where it is defined/?